### PR TITLE
API Happenings

### PR DESCRIPTION
**Pull request by afischer**
_It's the start of something new..._

Web scraping SDK for an athletics API. Implemented the "athlete," @adamdom is working on games and schedules.

Still todo: tests, many other things.

### CODE REVIEW

File: pennathletics/scraper.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        'SPSID' : 8606
+    }
+}
+
+
+ROSTER_URL = 'http://www.pennathletics.com/SportSelect.dbml?&DB_OEM_ID=1700&SPID={}&SPSID={}&Q_SEASON={}'
+
+# requests.get("http://www.pennathletics.com/SportSelect.dbml?&DB_OEM_ID=1700&SPID=540&SPSID=8650")
+
+def scrapeRoster(sport, year):
```
Comment (pbjr23) [inline pennathletics/scraper.py]: In accordance with https://www.python.org/dev/peps/pep-0008/#function-names, please avoid camelCase for function names

Comment (afischer) [inline pennathletics/scraper.py]: gurl.

File: pennathletics/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 __title__ = 'pennathletics'
 __version__ = '0.0.1'
-__author__ = 'Adel Qalieh'
+__author__ = 'Adel Qalieh, Andrew Fischer'
```
Comment (adelq) [inline pennathletics/__init__.py:3]: We should get rid of this and replace with `Penn Labs` and have a separate `AUTHORS.rst` in the vein of [requests](https://github.com/kennethreitz/requests/blob/master/AUTHORS.rst)

File: pennathletics/athletes.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            person[2],
+            parse.ht_feet_to_inches(person[3]),
+            person[4],
+            person[5],
+            parse.parse_hometown(person[6]))
+        )
+    return roster
+        
+    
+# print(get_roster('M_Basketball',2015))
```
Comment (adelq) [inline pennathletics/athletes.py:22]: Should have no comments in the final committed files, no biggie though

File: pennathletics/athletes.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function #PYTHON 3 BABY
+import models, parse
```
Comment (adelq) [inline pennathletics/athletes.py:2]: Please use something like `pyflakes` to lint your code. PEP8 is the universal python code style guide, only 1 import statement per line.

File: pennathletics/scraper.py [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    },
+    'W_Tennis' : {
+        'SPID'  : 553,
+        'SPSID' : 8943
+    },
+    'W_Volleyball' : {
+        'SPID'  : 538,
+        'SPSID' : 8606
+    }
+}
```
Comment (adelq) [inline pennathletics/scraper.py:123]: 1. You should have this in a separate .py file for all the miscellaneous hardcoded data.
2. You should be using [namedtuples](https://docs.python.org/2/library/collections.html#collections.namedtuple), not dictionaries.

File: pennathletics/scraper.py [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+        roster.append(parsed)
+    
+    roster = roster[7:] # remove header crap
+
+    # print(roster)
+    return roster
+
+    
+# print(SPORTS['W_Swimming']['SPID'])
+scrape_roster('M_Basketball',2015)
```
Comment (adelq) [inline pennathletics/scraper.py:153]: No code comments or test calls in the final committed version.

File: pennathletics/scraper.py [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+ROSTER_URL = 'http://www.pennathletics.com/SportSelect.dbml?&DB_OEM_ID=1700&SPID={}&SPSID={}&Q_SEASON={}'
+
+# requests.get("http://www.pennathletics.com/SportSelect.dbml?&DB_OEM_ID=1700&SPID=540&SPSID=8650")
+
+def scrape_roster(sport, year):
+    """Returns a list of lists contianing individual player information for a team.
+    :param sport: string value of sport.
+    :param year: 4 digit int of year.
+    """
```
Comment (adelq) [inline pennathletics/scraper.py:134]: love the docstrings

**Comment by adelq** [comment]
Added some code review comments, but as this is already merged they can be fixed in a future update.

File: pennathletics/scraper.py (hunk 5 above)
Comment (afischer) [inline pennathletics/scraper.py:123]: :+1: @TODO

**Pull request merged**